diary/logdb.py

```diff
@@ -69,6 +69,10 @@ def close(self):
         except sqlite3.ProgrammingError:
             pass
 
+    def destroy(self):
```

Comment:
That's really great to hear :)! Most of your messages were very terse, and I was worried that you didn't want me to contribute. Thanks for reassuring me.